setup.py

```diff
@@ -142,6 +142,10 @@ def copy_data_tree(datadir, destdir):
 gdal_major_version = int(gdal_version_parts[0])
 gdal_minor_version = int(gdal_version_parts[1])
 
+if gdal_major_version == 1 and gdal_minor_version < 11:
+    log.fatal("GDAL >= 1.11 is required for rasterio.  Please upgrade GDAL.")
+    sys.exit(1)
```

Comment:
Should this be an exception instead of an `exit()`?  Couldn't a bad logging configuration obscure the message?

Reply:
Good suggestion, I was just using the existing pattern in `setup.py`.  I moved all places where we were calling `sys.exit(1)` to `sys.exit("ERROR: some error message")`, since [Python turns anything other than an integer into a `sys.exit(1)` and prints the error message to `stderr`](https://docs.python.org/2/library/sys.html#sys.exit).

Raising an exception here has the downside of spitting out the full stack trace, which can obscure the actual error.  In this case, stack trace is irrelevant, so I wanted a very succinct error message only.